addons/api/mirage/factories/host.js

```diff
@@ -18,4 +45,16 @@ export default factory.extend({
       return this.hostCatalog.plugin;
     }
   },
+  attributes() {
+    return {
+      address: internet.ipv6(),
+      ip_addresses: returnIpAddressesArray(),
+      external_id: datatype.uuid(),
+      // Only aws plugins have dns_names
+      dns_names:
+        this.type != 'static' && this.plugin?.name === 'aws'
```

Comment:
do we need the `type` check here?

Reply:
Actually is redundant to checking plugin. Changing it